cypress/integration/workspace.test.ts

```diff
@@ -24,7 +24,6 @@ context("Test the overall workspace", () => {
 
       it("will verify that left nav area is closes when other tabs are opened", () => {
           // should this be tab closes when no longer in that area? my work and left nav
-          cy.visit("/?appMode=qa&fakeClass=5&fakeUser=student:1&fakeOffering=1&qaGroup=1&problem=1.1");
```

Comment:
Bonus! Eliminating this redundant visit should speed up the tests and may make them less likely to fail.

Reply:
:D that is the hope!